src/tests/functional/plugin/shared/include/behavior/plugin/core_integration.hpp

```diff
@@ -368,7 +368,7 @@ TEST_P(IEClassBasicTestP, ImportNetworkThrows) {
             file << "content";
         }
 
-        EXPECT_THROW(ie.ImportNetwork(modelName, target_device), InferenceEngine::NotImplemented);
+        EXPECT_THROW(ie.ImportNetwork(modelName, target_device), InferenceEngine::GeneralError);
```

Comment:
I think this tests is intended to check correct behavior for plugins which don't support Import/Export API. So if we support it now, then maybe whole test can be removed.

Reply:
I removed it as you reviewed. Thank you.